src/config/dynamic-theme-fixes.config

```diff
@@ -21293,6 +21293,45 @@ CSS
 
 ================================
 
+uottawa.brightspace.com
+
+CSS
+button:hover, button:focus, :host([active]) button {
```

Comment:
```suggestion
button
:host([active]) button {
```

Reply:
Hmm, removing the hover/focus selectors makes the changes not apply to the button when you hover over it.

Reply:
The `!important` on the CSS declaration should take care of that..

Reply:
Oh, changing the selector to :host works (the site has an obsession with shadow dom).

Reply:
Ok, nonetheless I have added the `!imporatant` statements. Sorry about that.